### DISCUSSION TITLE
Account Flagged for a Month- No Response from Support

### DISCUSSION THREAD

**Discussion by damibruneteqaunti** [community/community#108151] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It's been a one month since my friend's GitHub account got flagged, and despite reaching out to support, we haven't received any response.

Upon checking the GitHub Security log, we noticed login activity from an unfamiliar country, indicating a possible hack. We promptly changed the password, but the account remains flagged.

Support is not responding. What should be our next steps? Any advice would be greatly appreciated.

His account: @Generalsimus
Support ticket: #2586351

**Comment by estrosebas** [comment]
same problem, still no solution

**Comment by andrii-zhygalko** [comment]
Same issue, account @zhyndr suddenly got flagged.
The reinstatement request #2594059 has been opened but there is no any answer after more than a week.
That account is used for study (a course in web development).
If someone from the github staff reads this, please help us. Thank you!

**Comment by Jaic1** [comment]
Same issue, my friend's account @zhangt2333 got flagged dut to password randomized.

The password has been changed and 2FA has also been enabled.

The reinstatement request [#2570849](https://support.github.com/ticket/personal/0/2570849) has been opened but there is no any answer after 20 days.

If someone from the github staff reads this, please help us. Thank you!

**Reply by zhangt2333** [reply]
My account have been unflagged 5 minutes ago, thanks :)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @damibruneteqaunti! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, opening a support ticket is the best course of action for issues like this. Here are some resources on the process of [Creating a Support Ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket#about-support-tickets) and [How Ticket Priority is Determined](https://docs.github.com/en/enterprise-cloud@latest/support/learning-about-github-support/about-ticket-priority).

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by damibruneteqaunti** [reply]
Thank you for your response.

I actually submitted a support ticket over two weeks ago, but unfortunately, I have yet to receive a reply.

My ticket: #2586351

**Reply by ghostinhershell** [reply]
Hey @damibruneteqaunti, 

It looks like you submitted thw support ticket on behalf of your friend. For security reasons, your friend will need to submit a ticket to support using the email address associated with their GitHub account in order to move forward. We cannot move forward until your friend reaches out to support themselves.

**Reply by Nora-rad** [reply]
@ghostinhershell  I don’t understand what happened to my account.
The reinstatement request #2678270 has been opened but there is no answer after 2 weeks. Would you tell me what to do now?

**Reply by asmawi117** [reply]
can you please flagging this to support team @ghostinhershell i have ticket open more than 3 months already #2593578

**Reply by hegdepavankumar** [reply]
Anyone please help me to get rid of this flagged issue, it's been more than 2-3 weeks such a horrible experience with GitHub. @hegdepavankumar  The reinstatement request # [#3316094]. anyone please help me. my all contribution and production account no access on public. 

Could you please take a look:  @ghostinhershell

**Comment by bkpaccountwilldeletelater** [comment]
I also had my account 'phalkmin' flagged last monday, it's almost 2 weeks now, not a single answer from support. I had this account for more than 10 years, and without any reason or warning, I got suspended while approving PRs - It's been a huge nuisance since then

**Reply by Soraiko** [reply]
Github think they're God the Father and they have the right to punish users forever like that. Like they never made mistakes nor suffered for being ignored by a support once and they can be so condescendant.

**Comment by AstralCoder581** [comment]
My account has been flagged and I have raised tickets and till now I haven't had any response back from github support. Please look into the issue. I am Having some important works to do which is where I am unable to make a pr.

**Comment by ahmeddotgg** [comment]
same thing here for a month, am i dumb for closing the ticket and creating a new one? 🤦‍♂️

**Comment by damibruneteqaunti** [comment]
After one month, the support team responded and recovered the account.

**Reply by Notmeher** [reply]
What was the reason behind flagged ? Did you get to know?

**Reply by bkpaccountwilldeletelater** [reply]
Meanwhile, it's more than one month and I'm still being ghosted by support :(

**Comment by cronta** [comment]
Same issue here
no response from the support

Account: willnaoosmith

**Comment by Notmeher** [comment]
Same issue for account: Notmeher 
Idk how long it will take for them to response!

**Comment by shure-dev** [comment]
I have also the same issue. Can I send an email? or call?

Please help

**Comment by afrazkhan1407** [comment]
same issue account has been flagged and its been more then a week and no response from the github

**Comment by B-Ilyas** [comment]
3 week here and still no answer........

**Comment by alperbayram** [comment]
Hey,
It's been a one week since my friend's GitHub account got flagged, and despite reaching out to support, we haven't received any response.

username:@khayym
Support ticket: #2895652
https://support.github.com/ticket/personal/0/2895652

**Comment by alperbayram** [comment]
Dear GitHub Support Team,

It has been a week since my friend's GitHub account (username: @khayym) was flagged. Despite reaching out to support, we have not received any response regarding the status of the issue (Support ticket: #2895652).

This situation is significantly impacting our ability to complete our projects. We would greatly appreciate it if you could expedite the review process and provide us with an update as soon as possible.

username:@khayym
Support ticket: #2895652

**Comment by zly2006** [comment]
Hi there,

I am under the same situation. My ticket is #2927229

Thanks so much if you could review the ticket. I have to use Pull Requests and collaborate with others.

**Reply by satyam-mishra-pce** [reply]
Hey, how did you resolve your issue? How much time did they take? I created a ticket a month ago... and no replies yet.

**Comment by Nexus20060** [comment]
My Account was flagged for spam and.. i don't know why. I created my Ticket on 4th June 2024 and still no response :/ . Ticket ID: #2824707

I hope i get a response "soon". 

edit: i got my account back after 1 year and 1 month of waiting

**Reply by swoboda1337** [reply]
1 year???? that is crazy
